scripts/benchmark_db/README.md

```diff
@@ -1,49 +1,114 @@
 # Forest benchmark db script
 
-This script is here to help with testing of Forest db backends and their
-configuration.
+This script was developed to help with testing of Forest db backends and their
+configuration; the script now also allows benchmarking ("daily" benchmarks) of
+Forest and Lotus snapshot import times (in min:sec) and validation times (in
```

Comment:
The reasoning might be more straightforward if we normalize it all to `seconds`.